supermarq-benchmarks/supermarq/qcvv/irb_test.py

```diff
@@ -82,9 +81,14 @@ def test_irb_random_clifford() -> None:
 
 
 def test_random_two_qubit_clifford() -> None:
```

Comment:
There's [too many mocks](https://github.com/Infleqtion/server-superstaq/issues/1244) going on here. Let's delete this test. There are quality end-to-end tests that are getting us the 100% coverage we need.